### PR TITLE
I18n core implementation 

### PR DESCRIPTION
**Pull request by darkfadr**
Feature sa rezoud #1

### CODE REVIEW

File: components/BackgroundImage.js [hunk 1]
```diff
@@ -1,16 +1,31 @@
 import React from "react";
-import "./BackgroundImage.scss";
+import styled from '@emotion/styled';
+// import "./BackgroundImage.scss";
```
Comment (accimeesterlin) [inline components/BackgroundImage.js]: Can we remove this comment?

Comment (darkfadr) [reply components/BackgroundImage.js]: Sure thing removed. Good catch

File: components/Footer.js [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 import Link from "next/link";
 import "./Footer.scss";
 
+const Copyright = styled.p`
+  margin-top: 2rem;
+  color: hsl(0, 0%, 50%);
+`;
+
 function Footer(props) {
+  const {color, logo, size, description, copyright} = props;
```
Comment (accimeesterlin) [inline components/Footer.js]: Can we add some space here?

Comment (darkfadr) [reply components/Footer.js]: spaces have been added

**Pull request merged**